examples/web-ui/material/menus/menu.smash

```diff
@@ -0,0 +1,18 @@
+* Navigate to first level page {{page}}
+    setFirstLevelMenuItems +? {
```

Comment:
I'm against having a step whose name is camelCased. It may help your editor's autocomplete, but it's harder to read. Let discuss this some more.

Also, you can attach a code block directly to the first line, making the `+?` unnecessary.